### ISSUE TITLE
Crashing again when writing primitives with certain models

### ISSUE THREAD

**Issue by Levedi** [mikeoverbay/TankExporter#79]
This is the error:

` <Data>!_Tank Exporter.exe</Data> 
  <Data>1.0.0.130</Data> 
  <Data>6428911b</Data> 
  <Data>KERNELBASE.dll</Data> 
  <Data>10.0.19041.2728</Data> 
  <Data>8aa4a53c</Data> 
  <Data>e0434352</Data> 
  <Data>0012e4f2</Data> 
  <Data>76c</Data> 
  <Data>01d966500a715b00</Data> 
  <Data>C:\Program Files (x86)\Coffee_\Tank Exporter\!_Tank Exporter.exe</Data> 
  <Data>C:\Windows\System32\KERNELBASE.dll</Data> 
  <Data>db9088dc-95d1-4fd2-ace5-c8edef316e27</Data> 
  <Data /> 
  <Data /> 
`

**Comment by mikeoverbay** [comment]
Did you try version 130?
This error is hard to follow in the script. Doesn't look like my code at all but..
It looks like some crash when its building the tree lists of the tanks.
There was a bug of running out of space. I didn't expect the list to grow this big in tier 8.

What OS are you running. I am assuming windows.
If a exception window pops up, paste the text from it here.

Let me know.. I try and fix it.

**Comment by Levedi** [comment]
I was using v.130, there wasn't any window popped up with the error, I looked in Event Viewer and there was the error
![image](https://user-images.githubusercontent.com/83945213/230580966-a8d3c94e-3e67-48ac-9a24-bac512d2565a.png)
Now i got this error
and this what i've got last time
![image](https://user-images.githubusercontent.com/83945213/230581324-2c4d8156-96d9-4f5f-8825-b3b16f1fdd18.png)

**Comment by mikeoverbay** [comment]
Hi..
I have been working on this. Sorry for the delay.
Give version 134 a try.
